.github/CONTRIBUTING.md

```
@@ -5,6 +5,102 @@ Please follow these guidelines to ensure a smooth contribution process.
 
+# Development Workflow
+
+1. Clone the repository
+
+  ```bash
+  git clone https://github.com/docker-seminar/docker-tutorial.git
+  cd docker-tutorial
+  ```
+
+2. Install dependencies
+  1. Install server dependencies
```

markdownlint-cli2 (0.17.2)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 2

(MD005, list-indent)